data-model/fdo-type/digital-media/0.5.0/examples/digital-media-derivative-example.json

```diff
@@ -0,0 +1,402 @@
+          "schema:name": "DiSSCo Translator Service",
+          "ods:hasRoles": [
+            {
+              "@type": "schema:Role",
+              "schema:roleName": "processing service",
+              "schema:startDate": "2023-10-02T12:31:34.806Z"
+            }
+          ],
+          "ods:hasIdentifiers": [
+            {
+              "@id": "https://hdl.handle.net/TEST/XXX-XXX-XXX",
+              "@type": "ods:Identifier",
+              "dcterms:title": "handle",
+              "dcterms:type": "Handle",
+              "dcterms:identifier": "https://hdl.handle.net/TEST/XXX-XXX-XXX",
+              "ods:isPartOfLabel": false,
+              "ods:gupriLevel": "GloballyUniqueStablePersistentResolvableFDOCompliant",
+              "ods:identifierStatus": "Preferred"
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "ods:hasAgents": [
+    {
+      "@id": "https://orcid.org/0000-0002-5669-2769",
+      "@type": "schema:Person",
+      "schema:identifier": "https://orcid.org/0000-0002-5669-2769",
+      "schema:name": "Sam Leeflang",
+      "ods:hasRoles": [
+        {
+          "@type": "schema:Role",
+          "schema:roleName": "creator",
+          "schema:startDate": "2023-10-02T12:31:34.806Z"
+        },
+        {
+          "@type": "schema:Role",
+          "schema:roleName": "metadataCreator",
+          "schema:startDate": "2022-04-03T11:01:23.843Z"
+        }
+      ],
+      "ods:hasIdentifiers": [
+        {
+          "@id": "https://orcid.org/0000-0002-5669-2769",
+          "@type": "ods:Identifier",
+          "dcterms:title": "ORCID",
+          "dcterms:type": "URL",
+          "dcterms:identifier": "https://orcid.org/0000-0002-5669-2769",
+          "ods:isPartOfLabel": false,
+          "ods:gupriLevel": "GloballyUniqueStablePersistentResolvable",
+          "ods:identifierStatus": "Preferred"
+        }
+      ]
+    }
+  ],
+  "ods:hasMediaDerivatives": [
+    {
+      "ac:accessURI": "https://api.dissco.eu/digital-media/v1/10.3535/XXX-XXX-XXX/thumbnail",
+      "dcterms:rights": "https://creativecommons.org/licenses/by/4.0/",
+      "dcterms:created": "2025-11-04T13:38:34.806Z",
+      "dcterms:type": "StillImage",
+      "dcterms:format": "image/jpeg",
+      "ac:subtype": "http://rs.tdwg.org/acsubtype/values/Photograph",
+      "ac:subtypeLiteral": "Photograph",
+      "dcterms:title": "Thumbnail version of https://doi.org/10.3535/XXX-XXX-XXX",
+      "dcterms:description": "This is a Thumbnail version of the Digital Media object https://doi.org/10.3535/XXX-XXX-XXX",
+      "ac:variant": "http://rs.tdwg.org/acvariant/values/v001",
+      "ac:variantLiteral": "Thumbnail",
+      "exif:PixelYDimension": 2000,
+      "exif:PixelXDimension": 1000,
+      "ods:hasAgent": [
+        {
+          "@id": "https://hdl.handle.net/20.5000.1025/ABC-DEF-GHI",
+          "@type": "schema:SoftwareApplication",
+          "schema:name": "DiSSCo Image Thumbnail Service",
+          "ods:hasIdentifiers": [
```

Comment:
do we need a new role for this kind of agent?

Reply:
Hmm Role's don't have a controlled vocab yet so we can do whatever we want. But it seems I forgot the role in the example. Shall we do processing-service as the others?

Reply:
That sounds good :+1: